### PR TITLE
chore: baddie refactor

### PR DESCRIPTION
**Pull request by Jamesbarford**

### CODE REVIEW

**Comment by wattsit** [comment]
Changes  to the `Tileset_V1.tres` file?

**Comment by wattsit** [comment]
image

Parse errors when opening the branch.

File: Moonshot/baddies/Attributes.gd [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+		"sprite": sprite,
+		"velocity": velocity,
+		"floor_vector": floor_vector,
+		"body": body,
+		"animation": animation,
+		"damage_to_player": damage_to_player,
+		"should_damge_on_collision": should_damge_on_collision,
+		"shot_speed": shot_speed,
+		"shot_damage": laser_damage,
+		"item_drop": item_drop,
```
Comment (whitew1994WW) [inline Moonshot/baddies/Attributes.gd:49]: laser_damage needs to be defined in the class attributes to make it work 👍  :)

**Review by whitew1994WW — CHANGES_REQUESTED** [review]
Reset the TileSet and add the laser damage (see comment) and all good with me

File: Moonshot/baddies/Attributes.gd (hunk 1 above)
Comment (whitew1994WW) [reply Moonshot/baddies/Attributes.gd:49]: Oh lol Bill already requested this change xD

**Comment by Jamesbarford** [comment]
> Changes to the `Tileset_V1.tres` file?

@wattsit @will-white-dativa it keeps changing for stel and I when you run the game. Even on `DEV`

File: Moonshot/baddies/Attributes.gd (hunk 1 above)
Comment (Jamesbarford) [reply Moonshot/baddies/Attributes.gd:49]: `"shot_damage": shot_damage`

**Comment by Jamesbarford** [comment]
Have reset

**Pull request merged**